tests/test_other.py

```diff
@@ -2543,21 +2542,24 @@ def test_file_packager(self):
     self.assertNotContained('below the current directory', proc2.stderr)
 
     def clean(txt):
-      return [line for line in txt.split('\n') if 'PACKAGE_UUID' not in line and 'loadPackage({' not in line]
+      lines = txt.splitlines()
+      lines = [l for l in lines if 'PACKAGE_UUID' not in l and 'loadPackage({' not in l]
+      return ''.join(lines)
 
-    assert clean(proc.stdout) == clean(proc2.stdout)
+    self.assertTextDataIdentical(clean(proc.stdout), clean(proc2.stdout))
 
     # verify '--separate-metadata' option produces separate metadata file
     os.chdir('..')
 
     run_process([PYTHON, FILE_PACKAGER, 'test.data', '--preload', 'data1.txt', '--preload', 'subdir/data2.txt', '--js-output=immutable.js', '--separate-metadata'])
-    assert os.path.isfile('immutable.js.metadata')
+    self.assertExists('immutable.js.metadata')
     # verify js output file is immutable when metadata is separated
     shutil.copy2('immutable.js', 'immutable.js.copy') # copy with timestamp preserved
+    time.sleep(3.0)
```

Comment:
Please add a comment justifying this, as in general a sleep like that looks very dangerous (but here it is indeed necessary, I agree).

Reply:
oops I certainly didn't mean to land that ..

Reply:
No wait, I think it's good? It ensures the timestamp would be different, so it forces the test to actually check, instead of by virtue of being fast enough having the same time.

Reply:
Is it worth ?   Isn't the really important thing that the contents don't change?   

Anyway I put it back with a comment.